packages/scene-composer/src/components/panels/SceneHierarchyPanel/SceneHierarchyPanel.tsx

```diff
@@ -35,7 +63,7 @@ const SceneHierarchyPanel = () => {
             </Toolbar>
           )}
           <Main>
-            <SceneHierarchyTree enableDragAndDrop={canReorderHierarchy === 'T1'} />
+            <SceneHierarchyTree enableDragAndDrop={canReorderHierarchy === 'T1'} nodesToExpand={nodesToExpand} />
```

Comment:
it should be better to trigger a function that expends all parent nodes instead of calculating an array when selection changed, so there is no direct relation between the expanded and selected node

Reply:
Changed the approach to handle auto expand. SceneHierarchyDataProvider is now maintaining a list of nodes from selected node to root and is updated when selection changes. This is used by the hierarchy tree/item to handle the default expand state.
User still has the ability to expand/collapse nodes manually using the tree view.